torch/csrc/jit/serialization/export.h

```diff
@@ -208,5 +208,7 @@ struct TORCH_API BytecodeEmitDefaultInputsGuard {
   bool prev_mode;
 };
 
+TORCH_API IValue to_tuple(std::vector<IValue> ivalues);
+TORCH_API IValue Table(const std::vector<std::pair<std::string, IValue>>& entries);
```

Comment:
Curious about the naming convention here: why `to_tuple` but not `to_table`?

Reply:
to_tuple is to wrap a vector of ivalues to a tuple (more generic). Table means to have pairs of "name": tuple and create a "named" table.